hooks/tk-multi-loader2/basic/scene_actions.py

```diff
@@ -157,16 +157,38 @@ def execute_action(self, name, params, sg_publish_data):
             raise IOError("File not found on disk - '%s'" % path)
 
         if name == _IMPORT:
-            self._import(path, is_sequence)
+            try:
+                self._import(path, is_sequence, sg_publish_data)
+            except Exception as e:
+                self.logger.exception("Unable to import %s: %s" % (path, e))
 
     ###########################################################################
     # helper methods
 
-    def _import(self, path, is_sequence):
+    def _import(self, path, is_sequence, sg_data):
         """
         Helper method to add the footage described by path to a comp
+
+        :param path: Path to the footage to add
+        :param is_sequence: Whether the footage is a sequence
+        :param sg_data: SG dictionary.
+        :returns:
         """
-        project = self.parent.engine.adobe.app.project
-        insertion_bin = project.getInsertionBin()
-        paths = [path.replace(os.path.sep, '/')]
-        project.importFiles(paths, 0, insertion_bin, is_sequence)
+        engine = self.parent.engine
+        current_project = engine.current_project
+        import_in_bin_template = self.parent.get_template("import_in_bin_template")
```

Comment:
Hmm, I'm really not sure we will want to share the same setting between breakdown2 and loader2... Too bad that this setting can't be hook based.